feed-drupal.xml

```diff
@@ -17,7 +17,8 @@
         <description>
           &lt;p&gt;
             {% if post.featured_image %}
-              &lt;img src=&quot;{{ site.url}}{{ post.featured_image }}&quot; alt=&quot;{{ post.featured_image_alt }}&quot; &gt;
+              {% capture img_directory %}/assets/img{% endcapture %}
+              &lt;img src=&quot;{{ site.url}}{{ img_directory }}{{ post.featured_image }}&quot; alt=&quot;{{ post.featured_image_alt }}&quot; &gt;
```

Comment:
missing space - change `{{ site.url}}` to `{{ site.url }}` and we're good to go 👍